src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -718,39 +731,49 @@ private Matches() {}
     return ua.getIsAirTransport();
   });
 
-  public static final Match<Unit> UnitIsArtillery = Match.of(obj -> {
-    final UnitType type = obj.getUnitType();
-    final UnitAttachment ua = UnitAttachment.get(type);
-    return ua.getArtillery();
-  });
+  public static Match<Unit> unitIsArtillery() {
+    return Match.of(obj -> UnitAttachment.get(obj.getUnitType()).getArtillery());
+  }
```

Comment:
There was some manual refactoring done here in addition to the field -> method conversion.